e2e-tests/tests/monitoring/01-assert.yaml

```diff
@@ -14,17 +14,7 @@ status:
   replicas: 1
   updatedReplicas: 1
 ---
+kind: Service
 apiVersion: v1
-count: 1
-involvedObject:
-  apiVersion: v1
-  kind: Service
+metadata:
   name: monitoring-service
-kind: Event
-message: Ensured load balancer
-reason: EnsuredLoadBalancer
-reportingComponent: ""
-reportingInstance: ""
-source:
```

Comment:
Why should we delete all this stuff?

Reply:
It's the same here: [percona/percona-server-mysql-operator@main/e2e-tests/tests/monitoring/01-assert.yaml](https://github.com/percona/percona-server-mysql-operator/blob/main/e2e-tests/tests/monitoring/01-assert.yaml) so I did it. Otherwise the test doesn't pass.

Reply:
@ptankov But wasn't this deleted only for the new helm chart?